src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -508,6 +508,8 @@ static constexpr struct
 } imageMimeTypes[] = {
     {"jpeg", sizeof("jpeg"), imageMimeTypeEncoding::base64},
     {"png", sizeof("png"), imageMimeTypeEncoding::base64},
+    {"webp", sizeof("webp"), imageMimeTypeEncoding::base64},
+    {"font/ttf", sizeof("font/ttf"), imageMimeTypeEncoding::base64},
```

Comment:
I think imageMimeTypes is a list of image types that can be assigned to an image element. Therefore, the `font` is not applicable because it is not suitable for image elements. I recommend to leave only webp. nice fix :)